### ISSUE TITLE
Binding classes to scoped slots

### ISSUE THREAD

**Issue by dmotors** [matfish2/vue-tables-2#348]
If I have a scoped slot such as this:

	<v-server-table url="getData" :columns="columns" :options="options" ref="myTable">
		<template slot="qty" scope="props">
			{{ props.row.get_item.qty }}
		</template>
	</v-server-table>

How would I be able to set a class for that column conditionally such as a v-bind:class?

I know there is a columnClasses option but that's pre-defined and not sure how I would manipulate that conditionally.

Basically I want to do something like if props.row.get_item.qty > 0 then set this class to change background color of the TD column.

I assume using javascript to manipulate dom would not be best practice unless it's only option.

**Comment by dmotors** [comment]
I was able to do a temporary workaround of wrapping it a div then set 1px height on the TD but seems like a hack to me:

	<template slot="qty" scope="props">
		<div v-if="props.row.get_item.qty > 0" class="bis">
			{{ props.row.get_item.qty }}
		</div>
		<div v-else class="oos">
			{{ props.row.get_item.qty }}
		</div>
	</template>

And then a class to color it:

	.bis {
		display:block;
		background-color: green;
		height: 100%;
		padding: 8px;
		vertical-align: middle;
	}

And here's the TD css:

	td.qty {
		height: 1px;
		padding: 0px;
		text-align: center;
	}

**Comment by dmotors** [comment]
Another user helped simplify it even more:

	<div :class='{bis: isGreaterThanZero(props.row.get_item.qty)}'>
